### ISSUE TITLE
Rest of POPL benchmarks

### ISSUE THREAD

**Issue by bennn** [migeed-z/retic_performance#20]
Convert POPL benchmarks to our format.
- [ ] how long will these take to run? Can finish in time?

**Comment by bennn** [comment]
They are:
- 13 benchmarks
- 1 module each
- less than 20 functions each

Should all finish in time, but let's get started very soon. tonight.

**Comment by bennn** [comment]
10 of the 13 benchmarks are running on karst

- need to fully type `nqueens`
- need to resize `go` and `pystone`, they're too many functions to run exhaustively

**Comment by bennn** [comment]
nqueens is running

**Comment by bennn** [comment]
`pystone` & `go` are running now.

Two caveats with `go`:
1. only the `Square` class is typed. This class has 5 methods (1 fields spec). If everything was typed there would be `2**37` configurations.
  - I picked Square because its methods were called the most of all the methods. (Counted via print, put a printf at the top of each method and counted the total)
2. the `Square` fields with object types are `Dyn`
  - using the "correct" types raised subtyping error, the error message was huge (because recursion)
  - I don't know a more-precise-than-Dyn alternative. Tried `object` , and `retic` doesn't know that as a type
